README.rst

```diff
@@ -0,0 +1,119 @@
+Stylist
+=======
+
+|BSD3 License| |GitHub release| |PyPI version| |GitHub merge testing|
+
+Stylist is a code style checking tool built on a framework which supports
```

Comment:
"code style checking" could maybe use some formatting i.e. code-style checking or code style-checking. If you disagree I understand though!